src/munged/conf.c

```diff
@@ -79,6 +79,7 @@
 #define OPT_TRUSTED_GROUP       269
 #define OPT_ORIGIN              270
 #define OPT_LAST                271
+#define OPT_LISTEN_BACKLOG      272
```

Comment:
Please move `OPT_LISTEN_BACKLOG` (with the value 271) above `OPT_LAST` (now with the last value of 272).

`OPT_LAST` is to be the final value. It was intended for range-checking, but I see now that code was never implemented.